### PR TITLE
Fix list indentation

### PR DESCRIPTION
**Pull request by jglick**
#6863 only makes it more obvious that the formatting here was wrong, without actually fixing it.

### Proposed changelog entries

* N/A

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6874"></a>

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Review by timja — APPROVED** [review]
ah I see what was meant by the comment now, thanks 😄 

> only makes it more obvious that the formatting here was wrong, without actually fixing it.

No plans to do changes in that PR other than the specifics and the auto formatting, improvements should be extracted out like this 😄

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
